lib/cpp/CMakeLists.txt

```diff
@@ -20,6 +20,7 @@
 # Remove the following once lib/cpp no longer depends on boost headers:
 include(BoostMacros)
 REQUIRE_BOOST_HEADERS()
+list(APPEND package_deps Boost)
```

Comment:
We do not have a runtime dependency on boost for the C++ runtime library any more, at least that's what I'm led to believe with the C++11 changes.  We need boost for building and for running unit tests, but libthrift itself doesn't link against boost any more.

Reply:
I will remove boost!